templates/helm/templates/cluster-role-binding.yaml.tpl

```diff
@@ -1,11 +1,22 @@
 apiVersion: rbac.authorization.k8s.io/v1
+{{ "{{ if eq .Values.installScope \"cluster\" }}" }}
 kind: ClusterRoleBinding
 metadata:
   name: {{ "{{ include \"app.fullname\" . }}" }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: ack-{{ .ServiceIDClean }}-controller
+{{ "{{ else }}" }}
+kind: RoleBinding
+metadata:
+  name: {{ "{{ include \"app.fullname\" . }}" }}
+  namespace: {{ "{{ .Release.Namespace }}" }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ack-{{ .ServiceIDClean }}-controller
```

Comment:
nit: Remove duplication by moving `apiGroup` and `name` to after `{{ end }}` (and removing from both conditional blocks)